tfjs-backend-webgpu/src/webgpu_util.ts

```diff
@@ -184,3 +186,17 @@ export interface WebGPULayout {
   bindGroupLayout: GPUBindGroupLayout;
   pipelineLayout: GPUPipelineLayout;
 }
+
+export type requiredLimitsNames =|'maxTextureDimension1D'|
```

Comment:
Is it better to move it to `device_limits.ts`? And only keep the names that we use for now. For example, `export type requiredLimitsNames = 'maxComputeWorkgroupsPerDimension';  We can add more in future as needed.

Reply:
Done.